config/routes.rb

```diff
@@ -7,6 +7,7 @@
       get "dishes/entrees", to: "dishes#entrees"
       get "dishes/appetizers", to: "dishes#appetizers"
       get "dishes/desserts", to: "dishes#desserts"
+      post "dishes/:id/rate", to: "dishes#rate"
```

Comment:
All of these can be nested under the dishes `resources` declaration. All the gets can be in a `collection` block and the `rate` can be in a `member` block.